### DISCUSSION TITLE
Removed PagerDuty v1 support code

### DISCUSSION THREAD

**Discussion by nsano-rururu** [jertel/elastalert2#116] [Q&A] [answered]
As of October 19, 2018, the V1 API is no longer supported, so I'd like to remove the v1 code, but please let us know what you think.

**Comment by ferozsalam** [comment, marked as answer]
The current ElastAlert default is to try the [v1 API](https://github.com/jertel/elastalert2/blob/master/elastalert/alerts.py#L1295). As a first step I think it would be nice to switch the default to v2, which gives people a way to identify and fix any alerts that won't work, without removing any functionality.

Then potentially in the release after that (or after some set time) we could drop the v1 API entirely.

**Reply by nsano-rururu** [reply]
>As of October 19, 2018, the V1 API is no longer supported, 

The v1 API is no longer supported. Did you read it?

**Reply by ferozsalam** [reply]
Yep - however, at my organisation we're currently using the v1 API for our ElastAlert integration and it seems to still work, even if it's unsupported.

I don't know how many people will be doing the same, but given that it is the default, I image many people will be in the same situation.

**Reply by jertel** [reply]
The V1 REST API is no longer functional, but the V1 Events API, which ElastAlert uses, is still available for use. So let's not make a change here.
